notebooks/test/executed_notebooks/BigQuery Basics_output.ipynb

```diff
@@ -0,0 +1,615 @@
+   "metadata": {},
+   "source": [
+    "## Create a new dataset\n",
+    "\n",
+    "A dataset is contained within a specific [project](https://cloud.google.com/bigquery/docs/projects). Datasets are top-level containers that are used to organize and control access to your [tables](https://cloud.google.com/bigquery/docs/tables) and [views](https://cloud.google.com/bigquery/docs/views). A table or view must belong to a dataset, so you need to create at least one dataset before [loading data into BigQuery](https://cloud.google.com/bigquery/loading-data-into-bigquery)."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 6,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import time\n",
+    "\n",
+    "# Define a name for the new dataset.\n",
+    "dataset_id = 'test_dataset_1548268897503'\n",
+    "\n",
+    "# Create a DatasetReference using a chosen dataset ID.\n",
+    "# The project defaults to the Client's project if not specified.\n",
+    "dataset_ref = client.dataset(dataset_id)\n",
+    "\n",
+    "# Construct a full Dataset object to send to the API.\n",
+    "dataset = bigquery.Dataset(dataset_ref)\n",
+    "# Specify the geographic location where the dataset should reside.\n",
+    "dataset.location = 'US'\n",
+    "\n",
+    "# Send the dataset to the API for creation.\n",
+    "dataset = client.create_dataset(dataset)  # API request"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Load data from a Pandas DataFrame to a new table"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 7,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Loaded dataframe to /projects/ajhamilton-scratch/datasets/test_dataset_1548268897503/tables/monty_python\n"
+     ]
+    }
+   ],
+   "source": [
+    "records = [\n",
+    "    {\"title\": \"The Meaning of Life\", \"release_year\": 1983},\n",
+    "    {\"title\": \"Monty Python and the Holy Grail\", \"release_year\": 1975},\n",
+    "    {\"title\": \"Life of Brian\", \"release_year\": 1979},\n",
+    "    {\"title\": \"And Now for Something Completely Different\", \"release_year\": 1971},\n",
+    "]\n",
+    "\n",
+    "# Optionally set explicit indices.\n",
+    "# If indices are not specified, a column will be created for the default\n",
+    "# indices created by pandas.\n",
+    "index = [\"Q24980\", \"Q25043\", \"Q24953\", \"Q16403\"]\n",
+    "df = pandas.DataFrame(records, index=pandas.Index(index, name=\"wikidata_id\"))\n",
+    "\n",
+    "table_ref = dataset_ref.table(\"monty_python\")\n",
+    "job = client.load_table_from_dataframe(df, table_ref, location=\"US\")\n",
+    "\n",
+    "job.result()  # Waits for table load to complete.\n",
+    "print(\"Loaded dataframe to {}\".format(table_ref.path))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Load query results to a table\n",
```

Comment:
I'd recommend putting this up by the query example and changing the heading to "Writing query results to a destination table". It's not a load job, so this heading seems out-of-place.

Reply:
Updated the title and moved it just below dataset creation, because they'll need a dataset first.